.github/workflows/ci.yml

```
@@ -75,45 +72,40 @@ jobs:
 
     - name: Codecov
       run: |
-        codecov -n "GitHub Actions - ${{ matrix.task.name}} - ${{ matrix.os.name }} ${{ matrix.python.name }}"
+        codecov -n "GitHub Actions - Test - ${{ matrix.os.name }} ${{ matrix.python.name }}"
 
   check:
-    name: ${{ matrix.task.name}} - ${{ matrix.python.name }}
+    name: Check
```

maybe rename `Static checks` ? 'Check is a  bit generic'
```suggestion
    name: Static checks
```

---

Seems counter to being short.  `:]`  Is there really not a single word that has the desired meaning?  It seems it only needs to be distinct from 'test'.